### ISSUE TITLE
"Can't read Kafka version from MANIFEST.MF"

### ISSUE THREAD

**Issue by uhoh-itsmaciek** [kvlou/zkkf#1]
Keep in mind this is the first time I'm working with Java in 3+ years and I never used `mvn`, so maybe I'm making stupid mistakes.

Following the instructions, I see an error in the log after startup:

```
20:10:34 kf.1   | 2015-04-20T20:10:34-0700 pri=WARN, t=main, at=k.c.AppInfo$ Can't read Kafka version from MANIFEST.MF. Possible cause: java.lang.NullPointerException 
```

Sure enough, there doesn't seem to be a version specified in any of the manifests:

```
maciek@mothra:~/code/aux/zkkf$ for jar in $(find -name \*.jar); do unzip -p $jar META-INF/MANIFEST.MF; done
Manifest-Version: 1.0
Archiver-Version: Plexus Archiver
Built-By: maciek
Created-By: Apache Maven
Build-Jdk: 1.8.0_40-internal
Main-Class: org.apache.zookeeper.server.quorum.QuorumPeerMain

Manifest-Version: 1.0
Archiver-Version: Plexus Archiver
Created-By: Apache Maven
Built-By: maciek
Build-Jdk: 1.8.0_40-internal

Manifest-Version: 1.0
Archiver-Version: Plexus Archiver
Built-By: maciek
Created-By: Apache Maven
Build-Jdk: 1.8.0_40-internal
Main-Class: kafka.Kafka

Manifest-Version: 1.0
Archiver-Version: Plexus Archiver
Created-By: Apache Maven
Built-By: maciek
Build-Jdk: 1.8.0_40-internal
```

This seems benign but I'm not really sure at this point.

**Comment by kvlou** [comment]
I have mostly tested this on OSX with Sun JDK 7. Will try on trusty with 8.

**Comment by kvlou** [comment]
fixed in 2b7ee1e
